data/json/monsters/mammal.json

```diff
@@ -245,6 +247,7 @@
     "fear_triggers": [ "SOUND" ],
     "placate_triggers": [ "MEAT" ],
     "death_function": [ "NORMAL" ],
+    "harvest": "animal_medium",
```

Comment:
Also will need to remove the FUR/BONES/FAT flags like you've done similarly on line 422